uppsrc/CtrlCore/X11Top.cpp

```diff
@@ -321,7 +321,8 @@ void TopWindow::Open(Ctrl *owner)
 	XSetClassHint(Xdisplay, GetWindow(), class_hint);
 	LLOG("WndShow(" << visible << ")");
 	WndShow(visible);
-	if(visible) {
+	Top *top = GetTop(); //aris002
+	if(top && visible) { //aris002
```

Comment:
The same use top and utop variables from Ctrl class instead...

Reply:
Same as https://github.com/ultimatepp/ultimatepp/pull/126#discussion_r1059701436